reana_server/utils.py

```diff
@@ -97,6 +99,21 @@ def get_fetched_workflows_dir(user_id: str) -> str:
     return tmpdir
 
 
+def get_quota_excess_message(user):
+    """Return detailed quota excess message."""
+    quota = user.get_quota_usage()
+    message = "User quota exceeded.\n"
+    for resource_type, resource in quota.items():
+        limit = resource.get("limit", {}).get("raw", 0)
+        usage = resource.get("usage", {}).get("raw", 0)
+        if 0 < limit <= usage:
+            resource_usage, _ = get_quota_resource_usage(resource, "human_readable")
+            message += f"Resource: {resource_type}, usage: {resource_usage}\n"
+
+    message += f"Please find more details here: {REANA_QUOTAS_DOCS_URL}"
```

Comment:
Instead of "Please find more details...", perhaps write simply: "Please see https://...". It is shorter and perhaps more inciting to take an action than just "find more information".

Reply:
(or "Please see ... to resolve the situation." or some such... you know what I mean)